safeeyes/config/lang/es.json

```diff
@@ -8,7 +8,7 @@
     },
     "exercises": {
         "short_break_close_eyes": "Cierra fuertemente tus ojos",
-        "short_break_roll_eyes": "Cierra tus ojos y mueve tus ojos en círculos con las manos",
+        "short_break_roll_eyes": "Cierra tus ojos y mueve tus ojos en círculos usando los dedos de las manos",
```

Comment:
Hi,
Google translates this as `Close your eyes and move your eyes in circles using the fingers of your hands`.  I've just noticed that it does not match the English version. Could you please provide the correct translation that matches with `Roll your eyes to each side` or `Roll your eyes left to right and vice versa`

Thanks